src/elements/TextInput.vue

```diff
@@ -61,20 +75,30 @@ const onChange = () => {
       <slot />
       <span v-if="required && model?.length === 0" class="required">*</span>
     </span>
-    <input
-      class="tbpro-input"
-      v-model="model"
-      :class="{'dirty': isDirty}"
-      :type="type"
-      :id="name"
-      :name="name"
-      :disabled="disabled"
-      :placeholder="placeholder"
-      :required="required"
-      @invalid="onInvalid"
-      @change="onChange"
-      ref="inputRef"
-    />
+    <span class="tbpro-input" :class="{ 'small-text': props.smallText }">
+      <span v-if="prefix" ref="inputPrefix" class="tbpro-input-prefix">{{
+        prefix
+      }}</span>
+      <input
+        class="tbpro-input-element"
+        v-model="model"
+        :class="{ dirty: isDirty }"
+        :type="type"
+        :id="name"
+        :name="name"
+        :disabled="disabled"
+        :placeholder="placeholder"
+        :required="required"
+        :maxLength="maxLength"
+        @invalid="onInvalid"
+        @change="onChange"
+        ref="inputRef"
+        :style="{
+          paddingLeft: inputPaddingLeft,
+          width: `calc(100% - ${inputPaddingLeft})`,
```

Comment:
@MelissaAutumn regarding your last comment, you mentioned replacing the width line with `paddingLeft: inputPaddingLeft`.

I'm guessing you mean removing this `width`, leaving only the `paddingLeft: inputPaddingLeft`?

Or did you mean a different width line?

Reply:
Yep, sorry that's the width line!

Reply:
Success! :tada: 
![image](https://github.com/user-attachments/assets/830d168b-86d4-4567-a3e5-bce6f93300ee)

Thank you, @MelissaAutumn  -- you were 100% correct. Removing the `width` and applying the `box-sizing: border-box ` fixed it

Reply:
Awesome, thanks both of you!